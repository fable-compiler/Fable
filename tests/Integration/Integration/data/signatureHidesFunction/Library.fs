module Lib

let g a = a + 1
let f b = g b