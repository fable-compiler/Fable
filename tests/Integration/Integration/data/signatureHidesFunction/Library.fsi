module Lib

val f: b:int -> int