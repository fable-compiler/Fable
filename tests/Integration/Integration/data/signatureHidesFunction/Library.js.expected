
function g(a) {
    return a + 1;
}

export function f(b) {
    return g(b);
}

